notifier/postgres/deletenotifications.go

```diff
@@ -0,0 +1,59 @@
+package postgres
+
+import (
+	"context"
+
+	"github.com/google/uuid"
+	"github.com/jackc/pgx/v4/pgxpool"
+	clairerror "github.com/quay/clair/v4/clair-error"
+	"github.com/rs/zerolog"
+)
+
+// deleteNotifications garbage collects notifications and their associated
+// id and receipt rows
+func deleteNotifications(ctx context.Context, pool *pgxpool.Pool, id uuid.UUID) error {
```

Comment:
Do you really need to delete notifications from the database? Isn't it better to mark it as processed and keep it in DB for future debugging in case something went wrong?

Reply:
If a notification is not delivered, it is marked undelivered and notifier will retry. 

After we successfully deliver however, we need to GC at some point, or we will always grow the DB unbounded.